smalltalksrc/VMMaker/InterpreterPrimitives.class.st

```diff
@@ -1821,36 +1821,6 @@ InterpreterPrimitives >> primitiveDivideLargeIntegers [
 
 ]
 
-{ #category : #'process primitives' }
-InterpreterPrimitives >> primitiveEnterCriticalSection [
```

Comment:
These we should keep, and we should use in the image to have a proper mutex implementation